packages/hvsock/hvsock.1.0.3/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "dave@recoil.org"
+authors: [ "David Scott" "Rolf Neugebauer" "Anil Madhavapeddy" "Simon Ferquel"]
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-hvsock"
+dev-repo: "https://github.com/mirage/ocaml-hvsock.git"
+bug-reports: "https://github.com/mirage/ocaml-hvsock/issues"
+doc: "https://mirage.github.io/ocaml-hvsock"
+
+build: [
+  [ "jbuilder" "subst"] {pinned}
```

Comment:
if you are using jbuilder you need to pass `"-p" name` to the subst command. See https://github.com/ocaml/opam-repository/pull/12279